compiler/common-artifacts/exclude.lst

```diff
@@ -5,6 +5,8 @@
 
 #[[ optimize : Exclude from circle optimization(circle2circle) ]]
 ## TensorFlowLiteRecipes
+optimize(UnidirectionalSequenceLSTM_000)
+optimize(UnidirectionalSequenceLSTM_001)
```

Comment:
Isn't it possible to optimize?

Reply:
I will check this. Thanks for pointing out.

Reply:
I found something to fix on `luci/import` and `tflchef` for running through circle2circle. after all fix it, I will update this file!